src/app/features/section/subpages/section-form/form/form.component.html

```diff
@@ -0,0 +1,47 @@
+<div class="form-wrapper">
+  <div>
+    <form [formGroup]="newSectionForm" (submit)="createSection()">
```

Comment:
zamiast (submit) ->   (ngSubmit)

"ngSubmit ensures that the form doesn’t submit when the handler code throws (which is the default behaviour of submit) and causes an actual http post request. '